### PR TITLE
[NEW COMPONENT] NavLink

### PR DESCRIPTION
**Pull request by coltonehrman**

### CODE REVIEW

File: components/NavLink.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  className?: string
+}
+
+const NavLink: React.FC<NavLinkProps> = ({
+  children,
+  path,
+  activePath,
+  as,
+  external,
+  className
```
Comment (songz) [inline components/NavLink.tsx:18]: `className=''`

File: components/NavLink.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}) => {
+  const active = path === activePath
+  const hasClass = typeof className !== 'undefined'
```
Comment (songz) [inline components/NavLink.tsx:21]:
```
if (path === activePath) {
  className += ' active'
}
```

Comment (coltonehrman) [reply components/NavLink.tsx:21]: I need the active variable for another line as well.

Comment (coltonehrman) [reply components/NavLink.tsx:21]:
```
 <a className={`${className} ${active ? 'active' : ''}`}>
   {active && <span className="sr-only">(current)</span>}
   {children}
</a>
```

**Pull request merged**